### PR TITLE
New architecture

### PR DESCRIPTION
**Pull request by ekazankova**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -18,21 +18,15 @@
     "strictNullChecks": false,
     "baseUrl": "./",
     "paths": {
-      "@store": [
-        "src/app/shared/store"
-      ],
-      "@store/*": [
-        "src/app/shared/store/*"
-      ],
       "@shared/*": [
```
Comment (ipakhomov) [inline tsconfig.json:21]: Could you please remove this alias?

File: app.config.ts [hunk 2]
```diff
@@ -1,108 +1,109 @@
...
+    // TODO: Configure this to use Sentry or remove
+    // hooks: {
+    //   postPublish: [
+    //     {
+    //       file: 'sentry-expo/upload-sourcemaps',
+    //       config: {
+    //         url: 'https://your-sentry-url',
+    //         organization: 'your-sentry-organization',
+    //         project: 'your-sentry-project',
+    //         authToken: 'your-sentry-token'
```
Comment (ipakhomov) [inline app.config.ts:99]: Could you please remove this? Now we store auth token in [env variable](https://docs.expo.dev/guides/using-sentry/#:~:text=token%20through%20the-,SENTRY_AUTH_TOKEN,-environment%20variable%20in) within project secrets

**Review by ipakhomov — CHANGES_REQUESTED** [review]

**Comment by ipakhomov** [comment]
@ekazankova as dicussed, we can include RTKQ migration from [your branch](https://github.com/RonasIT/react-native-starter/tree/ekazankova/rtk-query). Alternatively, you can open a separate PR in case this is more manageable for you

**Comment by ekazankova** [comment]
@ipakhomov RTKQ is in [this request](https://github.com/RonasIT/react-native-starter/pull/34)

**Review by ipakhomov — APPROVED** [review]
@ekazankova thanks!

**Pull request merged**